include/zenoh/api/session.hxx

```diff
@@ -863,11 +863,7 @@ class Session : public Owned<::z_owned_session_t> {
     }
 #endif
 
-#if defined(Z_FEATURE_UNSTABLE_API) && (defined(ZENOHCXX_ZENOHC) || Z_FEATURE_LIVELINESS == 1)
```

Comment:
#if defined(ZENOHCXX_ZENOHC) || Z_FEATURE_LIVELINESS == 1 is still needed because of pico

Reply:
fixed